### PR TITLE
Nt add index add scope

### PR DESCRIPTION
**Pull request by nooralnabi**
I added  scope active true and added index to client by name , contractor by contractor_id

### CODE REVIEW

File: db/migrate/20170630215909_add_index_to_clients.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddIndexToClients < ActiveRecord::Migration[5.0]
```
Comment (ssharba) [inline db/migrate/20170630215909_add_index_to_clients.rb]: @nooralnabi I would do one migration for both indexes.

File: app/models/contractor.rb [hunk 2]
```diff
@@ -4,4 +4,8 @@ class Contractor < ApplicationRecord
    validates_presence_of :name , :contractor_id, :client_id, :access_code
    validates_length_of :name , minimum: 2 , message: " it is too short"
```
Comment (ssharba) [inline app/models/contractor.rb]: @nooralnabi remove unnecessary validation please.

File: db/migrate/20170630215909_add_index_to_clients.rb (hunk 1 above)
Comment (nooralnabi) [inline db/migrate/20170630215909_add_index_to_clients.rb]: @ssharba you want to do rollback for this migration or both client and cotractoer

Comment (ssharba) [inline db/migrate/20170630215909_add_index_to_clients.rb]: just delete the second migration and move the code and merge it in one file

File: app/models/contractor.rb (hunk 2 above)
Comment (ssharba) [inline app/models/contractor.rb]: just remove validation on name.

**Pull request merged**